api/external/compliance/reporting/reporting.proto

```diff
@@ -1025,3 +1025,10 @@ message AssetListResponse {
 	//Assets list as per the filters applied
 	repeated Assets assets = 1;
 }
+message ComplianceConfig {
+    string policy_name = 1;
```

Comment:
We should not give user option to pass the policy name. He can pass an arbitrary name in the policy name which we won't be able to use.

Reply:
We should different APIs for every config or we need to assert the policies being passed.